README.md

```diff
@@ -1,134 +1,84 @@
 # GKE PoC Toolkit
 
 * [Introduction](#introduction)
-* [Pre-requisites](#pre-requisites)
-* [Deploy a Cluster](#deploy-a-cluster)
-* [Harden GKE Security](#harden-gke-security)
-* [Deploy Secure GKE Workloads](#deploy-secure-gke-workloads)
+* [Install](#install)
+* [Quickstart](#quickstart)
+* [What's next?](#what's-next)
 
 ## Introduction
 
-This toolkit sets out to provide a set of infrastructre as code (IaC) which deploys GKE with a strong security posture that can be used to step through demos, stand up a POC, and deliver a codified example that can be the basis of a production implementation your own CICD pipelines.
+The GKE PoC toolkit sets out to provide a set of infrastructure as code (IaC) which deploys GKE cluster with a strong security posture that can be used to step through demos, stand up a POC, and deliver a codified example that can be the basis of a production implementation in your own CICD pipelines.
```

Comment:
I wonder if we could add a sentence before this one, that describes in the simplest possible terms what this is. 

eg. `The GKE Proof of Concept (PoC) Toolkit is a demo generator for Google Kubernetes Engine (GKE). It sets out to provide...`

Reply:
I dig this